src/main/java/net/fabricmc/loom/extension/LoomGradleExtensionApiImpl.java

```diff
@@ -85,6 +86,7 @@ protected LoomGradleExtensionApiImpl(Project project, LoomFiles directories) {
 		this.transitiveAccessWideners = project.getObjects().property(Boolean.class)
 				.convention(true);
 		this.transitiveAccessWideners.finalizeValueOnRead();
+		this.intermediary = project.getObjects().property(String.class);
```

Comment:
Please use conventions to set the default value rather than getOrDefault when getting the URL.

Will do a better check when home as it's a bit hard on mobile